stream-chat-android-client/src/main/java/io/getstream/chat/android/client/notifications/LoadNotificationDataWorker.kt

```diff
@@ -56,8 +56,16 @@ internal class LoadNotificationDataWorker(
             val getChannel = client.queryChannel(channelType, channelId, QueryChannelRequest())
 
             val result = getChannel.zipWith(getMessage).await()
+
             if (result.isSuccess) {
                 val (channel, message) = result.data()
+                val messageParentId = message.parentId
+
+                if (messageParentId != null) {
```

Comment:
Might be a good idea to further optimize this in the future and don't fetch the parent message if it's already present in the DB

Reply:
Noting down as a further improvement.